pkg/connections/stomp/connection_test.go

```diff
@@ -48,9 +51,16 @@ func ListenAndServe(serverStarted chan bool) {
 	s := &server.Server{}
 	serverStarted <- true
 
+	fmt.Println("Starting a new STOMP testing server.")
+
 	err := s.ListenAndServe()
 	if err != nil {
 		fmt.Printf("Failed to open new STOMP testing server: %s\n", err.Error())
+
+		// We assume that the reason we can't open the internal server is because
+		// we have an exatenal one running.
```

Comment:
There are more robust ways but OK for now :)
s/exatenal/external/ typo.
can you modify the Printf above to say it's falling back to internal server?

Reply:
> There are more robust ways but OK for now :)

Can you make a PR fixing that ?